packages/storybook/package.json

```diff
@@ -111,7 +111,6 @@
     "ts-node": "^9.1.1",
     "typescript": "4.3.5",
     "uuid": "^8.1.0",
-    "webpack": "5.76.0",
-    "webpack4": "npm:webpack@^4.44.2"
```

Comment:
Can we remove webpack from the `common-versions.json` now? I think all uses of differing versions of webpack are now resolved..

Reply:
Will update and test.